content/chainguard/libraries/javascript/build-configuration.md

```diff
@@ -395,13 +495,13 @@ To change a project to use Chainguard Libraries for JavaScript, set the registry
 URL to point to your repository manager in your user `.yarnrc` file: 
```

Comment:
I'm confused by these instructions still being here. Shouldn't these refer to the `.npmrc` instead?